main.tex

```diff
@@ -19,5 +20,26 @@
 
 \maketitle
 
+% 2 in der Gliederung
+\section{Lebenszyklus der Initative}
+
+Nachdem nun die Ausgangslage geschildert wurde, wird in diesem Teil der historische Ablauf Internet.org's beschrieben, von der Einführung bis zum Verbot.
+
+\subsection{Übersicht}
+
+\definecolor{BGColor}{rgb}{1.0,1.0,1.0}
+\begin{timeline}{2015}{2017}{250}{145}
+  \MonthAndYearEvent{2}{2015}{Formeller Start}
+  \MonthAndYearEvent{4}{2015}{Partielle Einführung}
+  \MonthAndYearEvent{7}{2015}{Umbenennung zu Free Basics}
+  \MonthAndYearEvent{11}{2015}{Vollständige Einführung}
+  \MonthAndYearEvent{2}{2016}{Verbot}
+\end{timeline}
```

Comment:
Was haltet Ihr von dem Zeitstrahl?

Es gibt nicht all zu viele Optionen, und dieser sieht passabel aus. Wenn auch ein bisschen klein.

Reply:
Ich finde ihn in Ordnung 👍